tool.css

```diff
@@ -7,9 +7,20 @@ body {
 	color:white;
 }
 
+.mik{

```

Comment:
Really great work! Can you change 'mik' to something more project-specific, e.g - 'metasub-download'?

Reply:
Yes sure. I was just testing it first so I put what randomly came in my mind like always "mik". LOL

Reply:
:+1:

Reply:
Done. check it

Reply:
FYI, you can make subsequent changes in the same PR itself. I am deleting the old PRs as the new ones are merged.

Reply:
Ok I got it. Thanks